backup_portal_basedata.ps1

```diff
@@ -40,7 +40,7 @@ cd $PSScriptRoot
 
 # Set environment variables for the database
 $DEVENV_IP = '172.19.0.2'
-$BASETABLES='(agreement_categories|app_subscription_statuses|app_statuses|company_application_statuses|company_role_descriptions|company_roles|company_service_account_statuses|company_statuses|company_user_statuses|connector_statuses|connector_types|consent_statuses|countries|document_types|document_status|identity_provider_categories|invitation_statuses|languages|notifications|notification_type|use_cases)'
+$BASETABLES='(agreement_categories|app_subscription_statuses|app_statuses|company_application_statuses|company_role_descriptions|company_roles|company_service_account_statuses|company_statuses|company_user_statuses|connector_statuses|connector_types|consent_statuses|countries|document_types|document_status|identity_provider_categories|invitation_statuses|languages|notification_type|use_cases)'
```

Comment:
we certainly need to include table notifications in 22-portal-content.sql. To do so please update 'backup_portal_data.sh' and create corresponding powershell-script if you cannot run the sh-script on wsl